cve_bin_tool/checkers/ncurses.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for Ncurses
+
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-72/product_id-38464/GNU-Ncurses.html
+
+RSS feed: http://www.cvedetails.com/vulnerability-feed.php?vendor_id=72&product_id=38464&version_id=0&orderby=3&cvssscoremin=0
+
+"""
+from ..util import regex_find
+
+
+def guess_contains_ncurses(lines):
+    """Tries to determine if a file includes Ncurses
+    """
+    for line in lines:
+        if "NCURSES" in line:
```

Comment:
I'm not convinced this is the most unique string, since it seems like it might occur even if NCURSES was dynamically linked.  I don't suppose there were any better ones available?